### PR TITLE
Docs for better Next.js integration

### PR DESCRIPTION
**Pull request by robmadole**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -531,6 +533,96 @@ They are re-exported from both `@fortawesome/fontawesome-svg-core` and
...
+  render() {
+    const { Component, pageProps } = this.props
+    return <Component {...pageProps} />
+  }
+}
+
+export default MyApp
+```
+
+You may also wish to include you library calls in the `_app.js` code.
```
Comment (haf) [inline README.md]: your

File: README.md [hunk 2]
```diff
@@ -531,6 +533,96 @@ They are re-exported from both `@fortawesome/fontawesome-svg-core` and
...
+import App, { Container } from 'next/app'
+
+import { config, library } from '@fortawesome/fontawesome-svg-core'
+import '@fortawesome/fontawesome-svg-core/styles.css'
+config.autoAddCss = false
+
+import { faBars, faUser } from '@fortawesome/free-solid-svg-icons'
+import { faTwitter, faFacebook } from '@fortawesome/free-brands-svg-icons'
+
+library.add(faBars, faUser, faTwitter, faFacebook)
```
Comment (haf) [inline README.md]: Or just import them where they are used?

File: README.md (hunk 1 above)
Comment (robmadole) [reply README.md]: Got it. Thanks

File: README.md (hunk 2 above)
Comment (robmadole) [reply README.md]: I added a section below this to mention explicit import.

**Comment by robmadole** [comment]
Alright, thanks for the 👀 . Going to merge and we'll follow-up with more changes if needed.

File: README.md (hunk 2 above)
Comment (haf) [reply README.md]: I mean; I tried both in a nextjs project and there seems to be no upside to doing `library.add` at all?

File: README.md [hunk 3]
```diff
@@ -26,6 +26,8 @@
   * [Basic](#basic)
   * [Advanced](#advanced)
   * [TypeScript](#typescript)
+- [Integrating with other tools and frameworks](#integrating-with-other-tools-and-frameworks)
+  * [ext.js](#extjs)
```
Comment (Vadorequest) [inline README.md]: Next

Comment (robmadole) [reply README.md]: Doh. Thanks.

**Pull request merged**